rmgweb/database/templates/solvationSearch.html

```diff
@@ -97,7 +97,14 @@
       <td>{{form.solvent}}</td>
    </tr>
    <tr>
-      <th colspan="2", align="left"><p>Option 2. Temperature-dependent prediction</p></th>
+      <th colspan="2", align="left"><p> <br>
+         Option 2. Temperature-dependent prediction</p></th>
+   </tr>
+   <tr>
+      <th colspan="2", style="font-weight:normal", align="left", ><p>For details on the temperature-dependent method, please refer to <a href="http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/liquids.html?highlight=solvation">Solvation Thermochemistry</a> in documentation. <br>
+      Please cite our work in your publication: <br>
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Chung, Y. & Gillis R. & Green W. H. (2020). <i>Temperature dependent solvation free energy estimation from <br>
```

Comment:
Not sure if these `&nbsp` is necessary.

Reply:
I used `&nbsp` to add a tab space in front of the citation information so it looks more clear what portion of the text is the citation information. If you don't think it looks good, I can get rid of them

Reply:
I removed them.
27ed29a

Reply:
Thanks. Since when I look at the page in different page sizes. this part may looks weird sometimes. 
![image](https://user-images.githubusercontent.com/35771233/74774024-00fb5c80-5261-11ea-8a2b-93eb3dfc6d52.png)

Reply:
I moved this part to the top portion of the text as you suggested. 
4665f20